bin/main.rb

```diff
@@ -0,0 +1 @@
+puts 'Hello world!'
```

Comment:
The first line should be 
```ruby
#!/usr/bin/env ruby
```

Reply:
thanks for the suggestion :+1:  added the line and changed file permission